clients/src/main/java/org/apache/kafka/common/security/kerberos/KerberosShortNamer.java

```diff
@@ -98,4 +98,9 @@ public String shortName(KerberosName kerberosName) throws IOException {
         throw new NoMatchingRule("No rules applied to " + toString());
     }
 
+    @Override
+    public String toString() {
+      return principalToLocalRules.toString();
```

Comment:
It's useful to have a toString in any case, but I'd probably include the name of the class `KerberosShortNamer` in the `toString`.